webtau-graphql/src/main/java/org/testingisdocumenting/webtau/graphql/GraphQLReportDataProvider.java

```diff
@@ -53,15 +53,35 @@ public GraphQLReportDataProvider(Supplier<GraphQLCoverage> coverageSupplier) {
                 .map(GraphQLQuery::toMap)
                 .collect(Collectors.toList());
 
+        List<? extends Map<String, ?>> successBranches = coverageSupplier.get().coveredSuccessBranches()
```

Comment:
Nothing you introduced here and not to be addressed in this PR, more a note to ourselves, but we should really find a better way of handling the stuff in this function.  It's sooo repetitive.

Reply:
we can at least do a little better, agreed